src/utils/Logger.ts

```diff
@@ -0,0 +1,30 @@
+import { LogInterface } from "../interfaces/LogInterface";
+
+export class Logger implements LogInterface {
+  public debug(msg: string, ...supportingData: any[]): void {
+    this.emitLogMessage("debug", msg, supportingData);
+  }
+  public info(msg: string, ...supportingData: any[]): void {
+    this.emitLogMessage("info", msg, supportingData);
+  }
+  public warn(msg: string, ...supportingData: any[]): void {
```

Comment:
We should avoid using `any`. It's considered a warning in ESLint for now as there's existing cases that need addressing

Reply:
Think we can use `unknown` instead of `any`.